### ISSUE TITLE
[Follow up #20408] Fix Load new content behaviour correctly 

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#21403]
## Description 
[Follow up #20408] Fix Load new content behaviour correctly 

## Steps to reproduce 

   1. Follow steps from #20408 

## Actual result 

Tablet | Mobile
-- | --
<video src=https://user-images.githubusercontent.com/17010094/155658515-d0a081a6-0077-4bfe-bc8d-429dc58015e0.mp4> | <video src=https://user-images.githubusercontent.com/17885425/156076684-14281547-958c-4725-8f56-c160456967c2.mp4>

## Expected result
Jump to the top of feed but still show it

## Issue reproduces how often 
Easy

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? Yes with 1.36.x
- Can you reproduce this issue with the current Play Store Beta version? Yes
- Can you reproduce this issue with the current Play Store Nightly version? Yes

## Device details
   - Install type (ARM, x86): ARM
   - Device type (Phone, Tablet, Phablet): All
   - Android version: All
   
## Brave version
1.36.x onward

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards?NA
- Is the issue reproducible on the latest version of Chrome? NA

### Additional information

cc: @alexsafe @mattmcalister @Uni-verse

**Comment by kjozwiak** [comment]
@tapanmodh technically this should still be in the `1.39.x` milestone as the original PR landed in `1.39.x` and this wasn't uplifted yet. Once https://github.com/brave/brave-core/pull/12931 is merged into `1.38.x`, the release team will move the issue into the correct milestone, in this case, `1.38.x`. You can view/read the process via https://github.com/brave/brave-browser/wiki/Triage-Guidelines#uplifting.

**Comment by kjozwiak** [comment]
So as per the above, I created https://github.com/brave/brave-browser/issues/22182 and https://github.com/brave/brave-browser/issues/22183 as follow ups. The fix from https://github.com/brave/brave-core/pull/12852 has improved things on the phones but still seems broken on the tablet as per https://github.com/brave/brave-browser/issues/22183.

@brave/qa-team keep in mind the above when verifying the above issue. Going to label this as `android-phone-specific` as  https://github.com/brave/brave-core/pull/12852 didn't fix the tablet case. That will need to be fixed via https://github.com/brave/brave-browser/issues/22183.

**Comment by kjozwiak** [comment]
Above requires `1.38.87` or higher for `1.38.x` verification.

**Comment by Uni-verse** [comment]
Verified using `1.38.103, Chromium 100.0.4896.127` on `Samsung GS21` running `Android 12`

- Ensured `Load new content` will bring user to the top of news feed

https://user-images.githubusercontent.com/17885425/164326622-50715f12-b957-4515-a378-abdd3526ba1a.mp4
